atuin-client/src/import/zsh_histdb.rs

```diff
@@ -76,12 +82,9 @@ impl From<HistDbEntry> for History {
                     .to_string(),
             )
             .duration(histdb_item.duration)
-            .hostname(
-                String::from_utf8(histdb_item.host)
-                    .unwrap_or_else(|_e| String::from(""))
-                    .trim_end()
-                    .to_string(),
-            );
+            .exit(histdb_item.exit_status)
+            .session(format!("{}:{}", hostname, histdb_item.session))
```

Comment:
This isn't enforced (yet) by types, but by convention; Atuin uses UUIDs for the session ID. Specifically a uuidv7, but really any would work

Is it possible to ensure we do the same here?

The importer can group what it's importing by session, and then map that to a new session ID